cmd/commands/peersrpc_active.go

```diff
@@ -45,7 +45,7 @@ var updateNodeAnnouncementCommand = cli.Command{
 	Add or remove addresses where your node can be reached at, change the
 	alias/color of the node or enable/disable supported feature bits without
 	restarting the node. A node announcement with the new information will
-	be created and brodcasted to the network.`,
+	be created and broadcasted to the network.`,
```

Comment:
nit: think this should be `broadcast`, [link](https://www.merriam-webster.com/grammar/is-it-broadcast-or-broadcasted-forecast-or-forecasted-usage#:~:text=For%20example%2C%20%22broadcast%22%20is,%2C%20CNN%20broadcast%20a%20show.%22)

Reply:
yeah those parts are incorrect, feel free to fix them